src/components/tabs/themes/light/tabs.base.scss

```diff
@@ -4,7 +4,9 @@
 $tabs-animation-function: cubic-bezier(.35, 0, .25, 1);
 
 :host {
-    display: block;
+    display: flex;
+    flex-direction: column;
+    flex: 1;
```

Comment:
this equals to `1 1 0`, meaning that the `auto` is lost, I think setting only `flex-grow: 1` is a better alternative.

But, in any case, I don't understand why using this rule here which is not in the Angular component, that would mean different default behavior across platforms.

@simeonoff FYI, because we talked about this too.

Reply:
I am adding this to make the tabs component stretch by default, however, I just saw that the app builder is adding the `1 1 0` by default, so I will remove it.